src/components/Accordion/_accordion.scss

```diff
@@ -14,31 +14,69 @@
 
 // .cc-accordion-item {}
 
-// .cc-accordion-item__title {}
+.cc-accordion-item__title {
+  margin: 0;
+}
+
+.cc-accordion__content {
+  padding: var(--space-unit) 0;
+}
 
 .cc-accordion__button {
-  @extend %anchor-base;
   @extend %btn-base;
   @extend %btn-size-standard;
-  font-size: var(--accordion-button-text-size);
-  font-weight: 600;
-  text-transform: uppercase;
-  transition: color ease 0.4s;
-  
+
   .btn__icon {
     height: 1.5rem;
     width: 1.5rem;
   }
-  
+
   .btn__icon--right {
     margin-right: 0;
   }
 
-  .cc-accordion-item & svg {
-    transform: rotate(-90deg);
-  }
-
   .cc-accordion-item--active & svg {
     transform: rotate(0deg);
   }
 }
+
+.cc-accordion__button--chevron {
+  @extend %anchor-base;
+  font-size: var(--accordion-button-text-size);
+  font-weight: 600;
+  text-transform: uppercase;
+  transition: color ease 0.4s;
+}
+
+.cc-accordion__button--plus {
+  border-top: 1px solid var(--colour-grey-10);
+  display: flex;
+  font-size: var(--text-base-size);
+  height: auto;
+  justify-content: flex-start;
+  margin-right: -5px; // because we are rotating and svg by 45deg
+  padding-right: 5px; // the focus state doesn't look nice
```

Comment:
I am open to suggestions on how to fix it in a nicer way. 

This is a reason (an uneven focus state):

<img width="573" alt="Screenshot 2020-09-10 at 10 50 27" src="https://user-images.githubusercontent.com/10700103/92715878-f962e880-f355-11ea-8d0d-f0074143c6f7.png">

Reply:
If it was positioned absolutely presumably it wouldn't kink out like that?